test.sh

```diff
@@ -41,8 +37,12 @@ dlHttpFiles() {
 }
 
 dlTftpFiles() {
-  dir=${WORKSPACE}/build/static/tftp
-  if [ "${REPO_NAME}" != "on-tftp" ]; then
+  if [ -n "${MULTI}" ]; then
+    dir=${WORKSPACE}/on-tftp/static/tftp
+  else
+    dir=${WORKSPACE}/build/static/tftp
+  fi
+  if [[ "${REPO_NAME}" != *"on-tftp"* ]]; then
```

Comment:
why do we need the star (*) ? to support what kind of "$REPO_NAME" instance ? can you give an example ?